cardscan-base/src/main/java/com/getbouncer/cardscan/base/MachineLearningThread.java

```diff
@@ -21,21 +21,24 @@
 import java.util.ArrayList;
 import java.util.LinkedList;
 
-class MachineLearningThread implements Runnable {
-
-    class RunArguments {
-        private final byte[] mFrameBytes;
-        private final Bitmap mBitmap;
-        private final OnScanListener mScanListener;
-        private final OnObjectListener mObjectListener;
-        private final Context mContext;
-        private final int mWidth;
-        private final int mHeight;
-        private final int mFormat;
-        private final int mSensorOrientation;
-        private final float mRoiCenterYRatio;
-        private final boolean mIsOcr;
-        private final File mObjectDetectFile;
+public class MachineLearningThread implements Runnable {
+
+    protected class RunArguments {
```

Comment:
for my own understanding, can you add a brief comment to each of these constructors that indicates what it's used for?

Reply:
done